src/scss/skin-modern/components/_subtitleoverlay-cea608.scss

```diff
@@ -0,0 +1,29 @@
+// sass-lint:disable force-element-nesting
+.#{$prefix}-ui-subtitle-overlay.#{$prefix}-cea608 {
+  align-items: flex-end;
+  bottom: unset;
+  display: flex;
+  height: 86%;
+  justify-content: center;
+  left: unset;
+  margin: auto 10%;
+  pointer-events: none;
+  right: unset;
+  top: 50%;
+  transform: translate(0%, -50%);
+  transition: height $animation-duration-short ease-out;
+  width: 80%;
```

Comment:
If I remove with the letter spacing, as explained the  `font-size` required to get 32 character in 80% of the player width (the size of the box inside which cea-608 captions must be displayed).
It's too big meaning that there is no way to fit 15 lines inside 80% the the player height, especially, since we don't really have 80% of the player height available when the menu are shown. So two successive lines will overlap.

If I just reduce the `font-size` then the character width is not 1/32 of the total width and text is not properly right aligned.